src/directions/helpers.ts

```diff
@@ -62,3 +63,17 @@ export function coordinatesComparator(
     return a[0] === b[0] && a[1] === b[1];
   }
 }
+
+export function getWaypointsCoordinates(waypoints: Feature<Point>[]): [number, number][] {
+  return waypoints.map((waypoint) => {
+    return [waypoint.geometry.coordinates[0], waypoint.geometry.coordinates[1]];
+  });
+}
+
+export function getWaypointsBearings(waypoints: Feature<Point>[]): ([number, number] | undefined)[] {
+  return waypoints.map((waypoint) => {
+    return Array.isArray(waypoint.properties?.bearing)
+      ? [waypoint.properties?.bearing[0], waypoint.properties?.bearing[1]]
+      : undefined;
+  });
+}
```

Comment:
Please, add the doc-comments for the added methods. Look above for inspiration and examples

Reply:
Added